README.md

```diff
@@ -2,3 +2,32 @@
 
 Final project for Web delopment course (React development module).
 
+## for development
+In order to take part in development process you need to install the following packages besides **React**
```

Comment:
This packages is saved into `package.json`.  So no need in this part of readme. 🙂